marathon_acme/tests/test_fake_marathon.py

```diff
@@ -1,52 +1,44 @@
 import json
+from functools import partial
+from operator import methodcaller
 
-from testtools.matchers import Equals, Is
-from twisted.internet.defer import inlineCallbacks, DeferredQueue
-from twisted.protocols.loopback import _LoopbackAddress
-from txfake.fake_connection import wait0
+from testtools.assertions import assert_that
+from testtools.matchers import AfterPreprocessing as After
+from testtools.matchers import (
+    Equals, Is, MatchesAll, MatchesListwise, MatchesStructure)
+from testtools.twistedsupport import succeeded
 
 from marathon_acme.clients import (
-    HTTPClient, JsonClient, json_content, sse_content,
-    sse_content_with_protocol)
+    json_content, sse_content, sse_content_with_protocol)
 from marathon_acme.tests.fake_marathon import (
     FakeMarathon, FakeMarathonAPI, FakeMarathonLb)
-from marathon_acme.tests.helpers import fake_client, TestCase
 from marathon_acme.tests.matchers import (
     HasHeader, IsJsonResponseWithCode, IsMarathonEvent, IsSseResponse)
 
 
-class TestFakeMarathonAPI(TestCase):
-    event_requests = DeferredQueue()
+def collect_events(event_type, response):
+    data = []
+    sse_content(response, {event_type: data.append})
+    return data
```

Comment:
Maybe this (or another similar function) should take a list of event types and return a dict of event_type->list?

If so, that's probably a thing for a later "test cleanup" PR.